src/App.tsx

```diff
@@ -3,9 +3,12 @@ import { Footer } from "./components/Footer"
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import { NavBar } from './components/NavBar';
 import { Home } from './components/Home';
+
```

Comment:
New lines not needed :)

Reply:
I am just gonna have this comment here not on every extra one

Reply:
resolved